### ISSUE TITLE
Bug: Cannot see the stop sharing button when using Android's largest font and display size for accessibility

### ISSUE THREAD

**Issue by sleepyplacebo** [onionshare/onionshare-android#155]
- [ X] I have already searched the issue queue for my error message or symptom

## Details about your operating system

Operating system: Android 16

Version of Onionshare: 0.2.3-beta

## How did you install OnionShare

I installed via F-Droid.  

## What is the problem

After I start sharing a file with Onionshare, I am unable to see the stop sharing button or scroll down to it.  If I lower my font size to about half of the max limit, then the stop sharing button will show.  I need the largest Android font size and display size for accessibility reasons.  It would be nice if I didn't have to temporarily turn down the settings in order to use the stop sharing button.

**Comment by SaptakS** [comment]
This issues seems to be related to the Onionshare Android. Transfering it to the android repository.
